src/Pulp/BumpVersion.purs

```diff
@@ -43,17 +43,21 @@ checkPursPublish out = do
 -- | Returns the new version that we should bump to.
 bumpVersion :: Args -> AffN Version
 bumpVersion args = do
-  mbumpStr <- getOption "bump" args.commandArgs
-  mcurrent <- map (map snd) getLatestTaggedVersion
   out <- getOutputter args
-
-  case mbumpStr of
-    Just bumpStr -> do
-      -- TODO: this is gross. See also Pulp.Args.Types.versionBump
```

Comment:
This TODO comment is unrelated to what you're doing here, but it still applies so I'd rather keep it in